### PR TITLE
New navigation - ASM

### PR DESCRIPTION
**Pull request by drichards-87**
### What does this PR do? What is the motivation?

Updates ASM screenshots for new nav.

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: static/images/security/application_security/application-security-signal-2.png
Comment (aliciascott) [inline static/images/security/application_security/application-security-signal-2.png:1]: This screenshot will need the IP address starting in `172.` blurred, it's re-used in other places so just commenting once here.

File: static/images/security/application_security/blocked-true-2.png
Comment (aliciascott) [inline static/images/security/application_security/blocked-true-2.png:1]: We should probably blur the IP's listed here under Top IPs as well. The Client IP list is already blurred.

File: static/images/security/application_security/threats/trace_qualification/trace-qualification-traces_3.png
Comment (aliciascott) [inline static/images/security/application_security/threats/trace_qualification/trace-qualification-traces_3.png:1]: This one needs the Client IP's blurred.

**Review by aliciascott — APPROVED** [review]
This looks great, I tested the links and screenshots for accuracy and all are good! Just a few that need the IP's blurred, but non blocking so approved.

**Comment by drichards-87** [comment]
Closing PR as the requirements for the new nav were changed.

**Pull request closed without merging**